src/components/tutorials-sidebar/compositions/product-collections-sidebar.tsx

```diff
@@ -1,20 +1,22 @@
+import { LearnProductData } from 'types/products'
+import { CollectionCategorySidebarSection } from 'views/collection-view/helpers'
 import TutorialsSidebar, {
   SectionList,
   HorizontalRule,
   SectionTitle,
-} from 'components/tutorials-sidebar'
-import { LearnProductData } from 'types/products'
-import { CollectionCategorySidebarSection } from '../helpers'
+} from '../'
 
-interface CollectionViewSidebarProps {
-  product: LearnProductData
+export interface ProductCollectionsSidebarProps {
+  product: Pick<LearnProductData, 'name' | 'slug'>
   sections: CollectionCategorySidebarSection[]
+  isOverview?: boolean
 }
 
-export default function CollectionViewSidebar({
+export default function ProductCollectionsSidebar({
```

Comment:
This sidebar is shared between collection and product views so I lifted this up into the `tutorials-sidebar` dir within components, under a `compositions` directory. Super open to a new name there, or if it should live elsewhere. But I like the idea of keeping the iterations of the tutorial sidebar close to the main component files.